css/style.css

```diff
@@ -1,18 +1,522 @@
-.partners-list {
-	width: 100%;
-	margin-bottom: 20px;
-	border: 1px solid #000000;
-	border-collapse: collapse; 
-	background-color: #7de0d3;
+
+body{
+	font-family: "Bakbak One", "Roboto", sans-serif;
+	font-weight: 400;
+	font-style: normal;
```

Comment:
```
font-weight: 400;
font-style: normal;
```
is default values, you don't need to define them. But you need to set `font-size`.
Also you can use shorthand property `font` to set them in one line.

Reply:
1) does this apply to this font, because it just has no other values(font-weight, font-style), or does it apply to all fonts (that font-weight: 400; font-style: normal; )?
2) If I set the font-size here, it will apply to all places where it appears, won't it? Why would I do that if the font has different values throughout the site?
thanks in advance!

Reply:
1. All properties in css has default values. In our case it's `font-weight: 400;` `font-style: normal;`
[https://www.w3schools.com/cssref/pr_font_weight.php](https://www.w3schools.com/cssref/pr_font_weight.php)
[https://www.w3schools.com/cssref/pr_font_font-style.php](https://www.w3schools.com/cssref/pr_font_font-style.php)
2. Yes, font applies to whole document. In those places where the font will differ from the default, you indicate the required one.

Reply:
Please fix